server/views/taskList/eligibilityTask.pug

```diff
@@ -47,18 +47,30 @@ div.largeMarginBottom
 
                                 div#exceptionalCircumstances.pure-u-1-4.center.bold
                                     | Yes
-                        div.pure-g.borderTopLight.smallPaddingTopBottom
-                            div.pure-u-3-4.padRight15
-                                | Is there less than 4 weeks to go to the offender's CRD?
-                            if licenceStatus.decisions.excluded || unsuitable
+
+                        if licenceStatus.decisions.excluded || unsuitable
+                            div.pure-g.borderTopLight.smallPaddingTopBottom
+                                div.pure-u-3-4.padRight15
+                                    | Is there less than 4 weeks to go to the offender's CRD?
                                 div#crdTimeAnswer.pure-u-1-4.center.bold N/A
```

Comment:
not sure what the intent is above but display text doesn't relate to the if

Reply:
I think there are faults in this logic but I've done some changes in a different story and there's another story so just ignore it and we'll fix in those other stories